src/NServiceBus.AcceptanceTests/Sagas/When_handling_a_message_with_both_handler_and_timeout_handler.cs

```diff
@@ -0,0 +1,97 @@
+﻿namespace NServiceBus.AcceptanceTests.Sagas
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+
+    public class When_handling_a_message_with_both_handler_and_timeout_handler : NServiceBusAcceptanceTest
```

Comment:
I wouldn't be surprised if this would cause some length issues again. How about shortening it to `When_handling_message_with_handler_and_timeout_handler` ?